tests/ui/associated-types/associated-types-projection-bound-in-supertraits.rs

```diff
@@ -5,13 +5,13 @@
 // this case, the `Result=Self` binding in the supertrait listing of
 // `Int` was being ignored.
 
-trait Not {
+trait Not { //~ WARN trait `Not` is never used
```

Comment:
This test does not need to be run-pass. Can be converted to check-pass? Or build-pass?